pfrl/agents/goal_conditioned_td3.py

```diff
@@ -207,6 +248,13 @@ def update_policy_with_goal(self, batch):
         self.policy_loss_record.append(float(loss))
         self.policy_optimizer.zero_grad()
         loss.backward()
+        # get policy gradients
+        gradients = self.get_and_flatten_policy_gradients()
+        gradient_variance = torch.var(gradients)
+        gradient_mean = torch.mean(gradients)
+        self.policy_gradients_variance_record.append(float(gradient_variance))
+        self.policy_gradients_mean_record.append(float(gradient_mean))
+
```

Comment:
I'll inspect the values and shapes on my local pc.

Reply:
alright - it'll also print to the terminal when running (`get_statistics` is called)